### ISSUE TITLE
analyzer 'errors' (sic) on spread operator

### ISSUE THREAD

**Issue by davidmaxwaterman** [Polymer/polymer-analyzer#846]
Is this expected?

```
          this.set('_selected', { ...data, ...metaData });
                                  ~

src/view.html(447,38) error [parse-error] - Unexpected token ...
[21:48:27] 'build' errored after 12 s
[21:48:27] Error: 1 error(s) occurred during build.
    at BuildAnalyzer._done (.../node_modules/polymer-build/lib/analyzer.js:265:36)
    at BuildAnalyzer.<anonymous> (.../node_modules/polymer-build/lib/analyzer.js:225
```

I'm trying to use it from a gulpfile with a Polymer 1 project.

**Comment by usergenic** [comment]
I think this may be fixed at least as of analyzer 3.0.0-pre.8

**Comment by rictic** [comment]
Confirmed fixed at master: https://polymer.github.io/analyzer-playground/?input=%3Cbase+href%3D%22https%3A%2F%2Fpolygit.org%2Fpolymer%2B%3Amaster%2Fwebcomponents%2B%3Amaster%2Fshadycss%2Bwebcomponents%2B%3Amaster%2Fcomponents%2F%22%3E%0A%3Cscript+src%3D%22webcomponentsjs%2Fwebcomponents-lite.js%22%3E%3C%2Fscript%3E%0A%3Clink+rel%3D%22import%22+href%3D%22polymer%2Fpolymer.html%22%3E%0A%0A%3Chello-world%3E%3C%2Fhello-world%3E%0A%3Chello-world+place%3D%22Oakland%22%3E%3C%2Fhello-world%3E%0A%0A%3Cdom-module+id%3D%22hello-world%22%3E%0A++%3Ctemplate%3E%0A++++%3Cstyle%3E%0A++++++%3Ahost+%7B%0A++++++++display%3A+block%3B%0A++++++%7D%0A++++%3C%2Fstyle%3E%0A++++Hello+%7B%7Bplace%7D%7D%21%0A++%3C%2Ftemplate%3E%0A++%3Cscript%3E%0A++++window.addEventListener%28%27WebComponentsReady%27%2C+%28%29+%3D%3E+%7B%0A++++++if+%28false%29+%7B%0A++++++++import%28%27lol%27%29%3B%0A++++++++const+foo+%3D+%7B...%7B%7D%7D%3B%0A++++++%7D%0A++++++%2F**%0A+++++++*+The+starting+point+of+a+journey.%0A+++++++*%2F%0A++++++class+HelloWorld+extends+Polymer.Element+%7B%0A++++++++static+get+is%28%29+%7Breturn+%27hello-world%27%3B%7D%0A++++++++static+get+properties%28%29+%7B%0A++++++++++return+%7B%0A++++++++++++%2F**%0A+++++++++++++*+The+place+to+greet.%0A+++++++++++++*%2F%0A++++++++++++place%3A+%7B%0A++++++++++++++type%3A+String%2C%0A++++++++++++++value%3A+%27World%27%0A++++++++++++%7D%0A++++++++++%7D%3B%0A++++++++%7D%0A++++++%7D%3B%0A++++++customElements.define%28HelloWorld.is%2C+HelloWorld%29%3B%0A++++%7D%29%3B%0A++%3C%2Fscript%3E%0A%3C%2Fdom-module%3E%0A

**Comment by davidmaxwaterman** [comment]
How do I actually get this fix? I currently have polymer-build@1.6.0 which uses polymer-analyzer@2.7.0.

**Comment by usergenic** [comment]
We released all our tools yesterday and a hotfix this morning to polymer-cli.

If you `npm install polymer-build` (for example) you'll get version 3.0.0 which includes support for object spread and a host of other things.
